### ISSUE TITLE
Documentation of data and results reproducability

### ISSUE THREAD

**Issue by solyron** [ognjenkundacina/graph-neural-network-state-estimation#3]
"As an example, we uploaded the datasets generated with measurement variances of 10-5. Due to file size limits, we could not upload the training set with 10000 samples, so the test set MSE is slightly higher than the one reported in subsection A of the paper."

Please clarify the documentation of the data that was uploaded. In the paper it is stated that were applied a Gaussian noise variance of 10-5
for phasor measurements,10−3 for bus voltage magnitude and active and reactive power flow legacy measurements, and 10−1 for active and reactive
injection legacy measurements." and that
"For the IEEE 118-bus test case, we placed 500 legacy measurements and seven PMUs (i.e., seven bus voltage phasors and 26 branch current phasors) in each sample, resulting in 2.39 measurement redundancy"

If the uploaded data follows the last citation, what is the intention behind the first citation? 

2. Missing python files that generates test setups Robustness to Loss of Input Data and Behavior Under Malicious Data Injections can be provided?
